manifests/local-dind/dind-cluster-v1.10.sh

```diff
@@ -2401,7 +2401,7 @@ function dind::run_tiller {
     if [[ $? -eq 0 ]];then
         helm_version=$(helm version -c --template '{{.Client.SemVer}}')
         if [[ -n ${KUBE_REPO_PREFIX} ]];then
-            helm init --tiller-image ${KUBE_REPO_PREFIX}/tiller:${helm_version}
+            helm init --tiller-image ${KUBE_REPO_PREFIX}/tiller:${helm_version} --skip-refresh
```

Comment:
Why do you want to add this parameter ? Just want to skip downloading the stable repo's index.yaml file ? refer this [code](https://github.com/helm/helm/blob/master/cmd/helm/installer/init.go#L117-L128).

Reply:
Yes, just to skip the download of the index.yaml file on `https://kubernetes-charts.storage.googleapis.com`. It is blocked that you understand.